app/src/main/java/com/ecosystem/kin/app/MainActivity.java

```diff
@@ -346,6 +333,46 @@ private void createPayToUserOffer(String recipientUserID) {
 		}
 	}
 
+	private void showPayToUserDialog(final View v) {
+		PayToUserDialog dialog = new PayToUserDialog(v.getContext());
+		dialog.setOnDismissListener(new OnDismissListener() {
+			@Override
+			public void onDismiss(DialogInterface dialog) {
```

Comment:
it is confusing that you named this variable `dialog` when you already have one variable with that name